src/components/posts.js

```diff
@@ -22,23 +22,27 @@ const Post = ({ post }) => {
 				<h3>{post.frontmatter.title}</h3>
 			</Link>
 			<hr />
-			<p className={styles.articleInfo}>
-				<BsFillCalendarFill /> Date: {post.frontmatter.date}
-				{". "}
-				Time to read: {post.timeToRead}{" "}
-				{post.timeToRead > 1 ? "minutes" : "minute"}
-			</p>
+			<PostInformation
+				date={post.frontmatter.date}
+				timeToRead={post.frontmatter.timeToRead}
```

Comment:
Time to read is not part of the frontmatter

```suggestion
				timeToRead={post.timeToRead}
```